src/openrct2/platform/shared.c

```diff
@@ -122,6 +127,12 @@ void platform_update_palette(const uint8* colours, sint32 start_index, sint32 nu
 
 void platform_init()
 {
+#ifdef _WIN32 
+    LARGE_INTEGER freq;
+    QueryPerformanceFrequency(&freq);
```

Comment:
Since its platform specific I thought this is the best place, I used to call platform_init in core_init before I found it was commented somewhere else. I think the name fits and platform specific code shouldn't be part of the base code rather being independent. But I can move it if I must.